.buildkite/bk.integration.pipeline.yml

```diff
@@ -79,10 +104,12 @@ steps:
           - fleet-privileged
           - fleet-airgapped-privileged
 
-      - label: "arm:Group: {{matrix}}"
+      - label: "arm:sudo: {{matrix}}"
+        # TestOtelFileProcessing fails on ARM. Uncomment when fixed
```

Comment:
Is there a ticket tracking the failure that can be added to the comment?

Reply:
Already fixed here: https://github.com/elastic/elastic-agent/pull/6061